src/mpl2/src/pin_alignment.cpp

```diff
@@ -486,7 +490,8 @@ void SimulatedAnnealingCore::Initialize()
     delta_cost += abs(norm_cost_list[i] - norm_cost_list[i - 1]);
 
   delta_cost = delta_cost / (norm_cost_list.size() - 1);
-  init_T_ = (-1) * delta_cost / log(init_prob_);
+  //init_T_ = (-1) * delta_cost / log(init_prob_);
+  init_T_ =  100;
```

Comment:
Where did 100 come from?  It seems quite different than the prior approach.  Also remove the commented code if it truly isn't needed.